src/cdk/utils/create-lambda-integration.ts

```diff
@@ -50,6 +56,23 @@ export function createLambdaIntegration(
     }
   );
 
+  const secretsManagerPolicy = new aws_iam.PolicyStatement({
+    actions: ['secretsmanager:GetSecretValue'],
+    resources: [
+      'arn:aws:secretsmanager:' +
```

Comment:
Please use a template literal instead of string concatenation

Reply:
done